tests/deepks/Autotest1.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 # deepks_test executable path
-deepks_test=test_deepks
+deepks_test=abacus
```

Comment:
Autotest1.sh is used in unit test of deepks code. The executable should be test_deepks.